### PR TITLE
FE - mention @everyone

### PR DESCRIPTION
**Pull request by megrogan**

### CODE REVIEW

File: frontend/app/src/components/home/Markdown.svelte [hunk 1]
```diff
@@ -44,14 +46,23 @@
         });
     }
 
+    function replaceEveryone(text: string): string {
+        if (!text.includes("@everyone")) return text;
+        return text.replace(/(^|[\s(){}\[\]])@everyone($|[\s(){}\[\]])/g, (_match, _p1) => {
+            return " **@everyone** ";
```
Comment (hpeebles) [inline frontend/app/src/components/home/Markdown.svelte:52]: Is this meant to have the spaces on either side?

File: frontend/app/src/components/home/MessageEntry.svelte [hunk 2]
```diff
@@ -260,6 +260,8 @@
                 switch (userOrGroup.kind) {
                     case "user_group":
                         return `@UserGroup(${userOrGroup.id})`;
+                    case "everyone":
+                        return `@${userOrGroup.kind}`;
```
Comment (hpeebles) [inline frontend/app/src/components/home/MessageEntry.svelte:264]: Seems better to just return `@everyone`

File: frontend/app/src/components/home/Markdown.svelte (hunk 1 above)
Comment (megrogan) [reply frontend/app/src/components/home/Markdown.svelte:49]: yes

Comment (hpeebles) [reply frontend/app/src/components/home/Markdown.svelte:49]: Why? What if the message contains '(@everyone)', or starts /finished with @everyone?

File: frontend/app/src/components/home/Markdown.svelte [hunk 3]
```diff
@@ -44,14 +44,22 @@
         });
     }
 
+    function replaceEveryone(text: string): string {
+        return text.replace(/(^|[\s(){}\[\]])@everyone($|[\s(){}\[\]])/g, (_match, _p1) => {
+            return " **@everyone** ";
```
Comment (julianjelfs) [inline frontend/app/src/components/home/Markdown.svelte:49]: Shouldn't "everyone" be translated? Seems a bit odd to have a hard coded English word like that.

File: frontend/app/src/components/home/MessageEntry.svelte (hunk 2 above)
Comment (megrogan) [reply frontend/app/src/components/home/MessageEntry.svelte:264]: Nah

File: frontend/app/src/components/home/Markdown.svelte (hunk 3 above)
Comment (megrogan) [reply frontend/app/src/components/home/Markdown.svelte]: Nah

**Review by hpeebles — APPROVED** [review]

File: frontend/app/src/components/home/Markdown.svelte (hunk 3 above)
Comment (julianjelfs) [reply frontend/app/src/components/home/Markdown.svelte]: Why in the future? It's not like it's hard. It seems just obviously wrong to use a hard-coded English word in a translated system.

**Pull request merged**